src/BlazorWebView/src/Maui/Android/BlazorWebViewHandler.Android.cs

```diff
@@ -18,8 +19,26 @@ public partial class BlazorWebViewHandler : ViewHandler<IBlazorWebView, AWebView
 		private AndroidWebKitWebViewManager? _webviewManager;
 		internal AndroidWebKitWebViewManager? WebviewManager => _webviewManager;
 
+		private bool _loggerCreated;
+		private ILogger? _logger;
+
+		internal ILogger? Logger
+		{
+			get
+			{
+				if (!_loggerCreated)
+				{
+					_logger = Services!.GetService<ILogger<BlazorWebViewHandler>>();
+					_loggerCreated = true;
+				}
+				return _logger;
+			}
+		}
+
 		protected override AWebView CreatePlatformView()
 		{
+			Logger?.CreatingAndroidWebkitWebView();
+
```

Comment:
Is there a reason why we are creating this lazily?

Am I correct to assume that we believe `Logging` is not a hard dependency? (Inferred from the `GetService` call).

Would it make sense to add it as a dependency instead?